worker/src/RTC/DataProducer.cpp

```diff
@@ -224,11 +224,14 @@ namespace RTC
 
 				std::vector<uint16_t> subchannels;
 
-				subchannels.reserve(body->subchannels()->size());
```

Comment:
Isn't `subchannels` mandatory in FBS?

Reply:
No: https://github.com/versatica/mediasoup/blob/flatbuffers/worker/fbs/dataProducer.fbs#L44

It happened the same with `DataConsumer.send()`, TS creates a default empty array but the type is Optional and hence legit not to add it.